lib/gradebook.rb

```diff
@@ -1 +1,7 @@
+class Gradebook
+  attr_reader :name
+  def initialize(name)
+    @name =  name 
 
+  end
```

Comment:
The [assessment requirements](https://github.com/turingschool-examples/students_and_courses_2301#iteration-3) ask for this attribute and method to be called `instructor`. The attr_reader should also be `instructor` in order to create the method by that name. I'm not taking points off for it this time, but be aware of the details being asked of you. This would not be considered correct if it were the final assessment.
```suggestion
  def initialize(name)
    @instructor =  name 

  end
```